client/my-sites/plans-v2/translated-copy.tsx

```diff
@@ -0,0 +1,203 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { OPTIONS_JETPACK_SECURITY, OPTIONS_JETPACK_SECURITY_MONTHLY } from './constants';
+import {
+	PLAN_JETPACK_COMPLETE,
+	PLAN_JETPACK_COMPLETE_MONTHLY,
+	PLAN_JETPACK_SECURITY_DAILY,
+	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+	PLAN_JETPACK_SECURITY_REALTIME,
+	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
+} from 'lib/plans/constants';
+import {
+	PRODUCT_JETPACK_ANTI_SPAM,
+	PRODUCT_JETPACK_ANTI_SPAM_MONTHLY,
+	PRODUCT_JETPACK_BACKUP,
+	PRODUCT_JETPACK_BACKUP_DAILY,
+	PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY,
+	PRODUCT_JETPACK_BACKUP_REALTIME,
+	PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY,
+	PRODUCT_JETPACK_CRM,
+	PRODUCT_JETPACK_CRM_MONTHLY,
+	PRODUCT_JETPACK_SCAN,
+	PRODUCT_JETPACK_SCAN_MONTHLY,
+} from 'lib/products-values/constants';
+
+/**
+ * Type dependencies
+ */
+import type { SelectorProductCopy } from './types';
+
+export function getSelectorProductCopy(
+	productSlug: string,
+	translate: Function
+): SelectorProductCopy {
+	switch ( productSlug ) {
+		case PRODUCT_JETPACK_ANTI_SPAM:
+		case PRODUCT_JETPACK_ANTI_SPAM_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack Anti-spam' ),
+				shortName: translate( 'Anti-spam', {
+					comment: 'Short name of Jetpack Anti-spam',
+				} ),
+				tagline: translate( 'Block spam automatically' ),
+				description: translate(
+					'Automated spam protection for comments and forms. Save time, get more responses, and give your visitors a better experience.'
+				),
+				buttonLabel: translate( 'Get Anti-spam' ),
+			};
+		case PRODUCT_JETPACK_BACKUP:
+			return {
+				displayName: translate( 'Jetpack Backup' ),
+				shortName: translate( 'Backup', {
+					comment: 'Short name of the Jetpack Backup generic product',
+				} ),
+				tagline: translate( 'Recommended for all sites' ),
+				description: translate(
+					'Never lose a word, image, page, or time worrying about your site. {{em}}Available options: Real-time or Daily.{{/em}}',
+					{
+						components: {
+							em: <em />,
+						},
+					}
+				),
+				buttonLabel: translate( 'Get Backup' ),
+			};
+		case PRODUCT_JETPACK_BACKUP_DAILY:
+		case PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY:
+			return {
+				displayName: translate( 'Backup {{em}}Daily{{/em}}', {
+					components: {
+						em: <em />,
+					},
+				} ),
+				shortName: translate( 'Backup Daily', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Jetpack Backup Daily' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/move-plansv2-translates-from-constants-take-2)